grunt/tasks/browserify.js

```diff
@@ -25,6 +25,8 @@ module.exports = function() {
     entries: entries,
     debug: config.debug, // sourcemaps
     standalone: config.standalone, // global
+    builtins: {},
+    detectGlobals: false,
```

Comment:
I believe this incantation is necessary to stop browserify from bundling in a fake version of node's stream library.